search-relevancy-test/resources/test_run_history.csv

```diff
@@ -0,0 +1,25 @@
+Base Run,2017-07-12T17:02,2,1,V006 dataset should come before version 005,FAIL,Out of order,2 1,0.860,0.500,
+Base Run,2017-07-12T17:02,3,1,Soil moisture products should come before salinity products when searching for SMAP,FAIL,Out of order,2 1,0.860,0.500,
+Base Run,2017-07-12T17:02,4,1,Ancillary products should be below data products (can't reproduce locally),FAIL,Out of order,2 1,0.860,0.500,
+Base Run,2017-07-12T17:02,5,1,OMI products should be above TOMS products because OMI is a later generation.,FAIL,Out of order,2 1,0.860,0.500,
+Base Run,2017-07-12T17:02,6,1,Collections with Leaf area index in the titles should show up before others when searching for leaf area index,FAIL,Out of order,2 1,0.860,0.500,
+Base Run,2017-07-12T17:02,9,1,ABoVE project collections should be ahead of collections with above in the description,PASS,,1 2,1.000,1.000,
+Base Run,2017-07-12T17:02,10,1,NPP should return ecology results first,PASS,,1 2,1.000,1.000,
+Base Run,2017-07-12T17:02,11,1,SMAP soil moisture searches should return L2-L4 SMAP above salinity products,FAIL,Out of order,2 1,0.860,0.500,
+Base Run,2017-07-12T17:02,11,2,SMAP soil moisture searches should return L2-L4 SMAP above L1 products,FAIL,Out of order,2 1,0.860,0.500,
+Base Run,2017-07-12T17:02,11,3,SMAP soil moisture searches should return L2-L4 SMAP above SMAPVEX products,FAIL,Out of order,2 1,0.860,0.500,
+Base Run,2017-07-12T17:02,12,1,SMAP L3 freeze/thaw products before anything else when searching for SMAP freeze/thaw,PASS,,1 2,1.000,1.000,
+Base Run,2017-07-12T17:02,14,1,World glacier inventory should be a top 3 collection when searching for glacier,FAIL,Out of order,2 1,0.860,0.500,
+Base Run,2017-07-12T17:02,14,2,GLIMS should be a top 3 collection when searching for glacier,FAIL,Out of order,2 1,0.860,0.500,
+Base Run,2017-07-12T17:02,15,1,Northern Hemisphere EASE-Grid 2.0 Version 4 should be ranked high because temporal coverage is very long and community usage is high.,FAIL,Out of order,3 1 2,0.817,0.500,
+Base Run,2017-07-12T17:02,15,2,"MODIS/Terra Snow Cover 5-Min L2 Swath 500m V006 should be ranked high because Global spatial coverage and long temporal coverage, continuously updated, and community usage is high.",FAIL,Out of order,1 3 2,0.973,1.000,
+Base Run,2017-07-12T17:02,15,3,"NOTE: Found an L3 slightly different - MODIS/Terra Snow Cover Daily L2 Swath 500m V006 should be ranked high because Global spatial coverage and long temporal coverage, continuously updated, and community usage is high.",FAIL,Out of order,3 1 2,0.817,0.500,
+Base Run,2017-07-12T17:02,20,1,Sea ice extent,FAIL,Out of order,1 4 3 2,0.945,1.000,
+Base Run,2017-07-12T17:02,26,1,Collections from ASDC data center above other data centers,FAIL,Out of order,2 1,0.860,0.500,
+Base Run,2017-07-12T17:02,32,1,,FAIL,Out of order,2 1,0.860,0.500,
+Base Run,2017-07-12T17:02,33,1,,FAIL,Out of order,2 1,0.860,0.500,
+Base Run,2017-07-12T17:02,SUMMARY,,,FAIL,4 tests passed out of 21,,0.892,0.643,
```

Comment:
Local is for when you're debugging locally and running again and again, which is why it's in the gitignore. The history is checked into the repo so when the next person makes relevancy changes they can see your previous results and use that to compare. I have the base run in there because that is before any improvements.

So when you were done making an improvement, you'd do a run and check it in.